src/cascadia/TerminalSettingsModel/KeyMapping.idl

```diff
@@ -35,5 +35,7 @@ namespace Microsoft.Terminal.Settings.Model
 
         Microsoft.Terminal.Control.KeyChord GetKeyBindingForAction(ShortcutAction action);
         Microsoft.Terminal.Control.KeyChord GetKeyBindingForActionWithArgs(ActionAndArgs actionAndArgs);
+
+        Windows.Foundation.Collections.IMap<Microsoft.Terminal.Control.KeyChord, ActionAndArgs> FetchGlobalHotkeys();

```

Comment:
nit: Fetch is a weird verb here. It seems like this is more accurately modelled as a property getter, like `GlobalHotkeys` or `GlobalHotkeyMap`